src/ui/windows/TogglDesktop/TogglDesktop/ui/windows/MainWindow.xaml.cs

```diff
@@ -548,6 +548,8 @@ private void editPopupVisibleChanged(object sender, DependencyPropertyChangedEve
             if (editPopup.IsVisible == false && ReferenceEquals(this.activeView, this.timerEntryListView))
             {
                 this.timerEntryListView.Entries.DeselectCells();
+                //TODO: Find a better way to undo the TE highlighting on Edit view closing
+                timerEntryListView.Timeline.ViewModel.SelectedTEId = null;

```

Comment:
how about extracting an extension method?
```suggestion
                timerEntryListView.DeselectTimeEntries();
```